### DISCUSSION TITLE
There is already OLM why is it still developing rukpak

### DISCUSSION THREAD

**Discussion by fanhaouu** [operator-framework/rukpak#582] [Q&A] [answered]
There is already OLM why is it still developing rukpak?

**Comment by bparees** [comment, marked as answer]
Rukpak is/will be a part of OLM.  It is the next generation of how OLM will manage/deploy operator resources.  It is intended to be less opinionated about the resources an operator packages/owns/deploys and give more control to operator authors and to pluggable provisioning systems that can define their own packaging formats for operators.

**Comment by timflannagan** [comment]
Looks like Ben already got to this. Just to extend what he wrote, we're looking at separating the current OLM monolith into separate sub-projects that can be installed independently. The rukpak project is one of those sub-projects, and this division of responsibilities between sub-projects allows you to install a component like rukpak if you don't require dependency resolution for your cluster application needs.

**Comment by fanhaouu** [comment]
Thank you very much for your detailed answers.Rukpak is a nice project that has simpler architecture and easier to understand and deploy.
@bparees @timflannagan
